wiki/Tournaments/RDOC/2020/en.md

```diff
@@ -27,35 +27,422 @@ The **Royal Danish osu!Cup 2020** (***RDOC 2020***) is an ongoing Danish 1v1 dou
+  - [Saint Snow - SELF CONTROL!! [Insane]](https://osu.ppy.sh/beatmapsets/880205#osu/2046872)
+- Freemod
+  - [Haru - Flowering Night (Instrumental) [Lunatic]](https://osu.ppy.sh/beatmapsets/6952#osu/30726)
+  - [O-Life Japan - Yakujinsama no Couple Dance [Insane]](https://osu.ppy.sh/beatmapsets/16990#osu/60743)
+- Tiebreaker
+  - **[anubasu-anubasu - Mercury Prayer (Extend Mix) [MiaoFu x EijiKuinbii's Rebirth]](https://osu.ppy.sh/beatmapsets/603755#osu/1649874)**
+
+### Round of 32
+
+- NoMod
+  - [TWICE - CHILLAX [Shining Wave]](https://osu.ppy.sh/beatmapsets/975301#osu/2041389)
+  - [Loki - With Fire and Sword (2013) [Extra]](https://osu.ppy.sh/beatmapsets/1088528#osu/2276016)
+  - [yuikonnu - Uchouten Vivace [Vivace!]](https://osu.ppy.sh/beatmapsets/107232#osu/281069)
+  - [namapann - Desire Drive [Lunatic]](https://osu.ppy.sh/beatmapsets/42946#osu/136540)
+- Hidden
+  - [Muse - Hysteria [FRENZIED]](https://osu.ppy.sh/beatmapsets/1139821#osu/2380876)
+  - [FOLiACETATE - Heterochromia Iridis [Extra]](https://osu.ppy.sh/beatmapsets/228529#osu/631361)
+- Hardrock
+  - [sana - Kokuhaku Yokou Renshuu [Expert]](https://osu.ppy.sh/beatmapsets/275283#osu/679226)
+  - [Megurine Luka - Anderutse [Insane]](https://osu.ppy.sh/beatmapsets/25258#osu/85530)
+- DoubleTime
+  - [Kami no Mizo Shiri-tai - Koi no Shirushi [Insane]](https://osu.ppy.sh/beatmapsets/40080#osu/127391)
+  - [petit milady - Koi wa Milk Tea [Amai]](https://osu.ppy.sh/beatmapsets/364652#osu/800385)
+- Freemod
+  - [Meg & Dia - Are There Giants Too, in the Dance?[Crazy!]](https://osu.ppy.sh/beatmapsets/34666#osu/112643)
+  - [cYsmix - Peer Gynt [Ex]](https://osu.ppy.sh/beatmapsets/880487#osu/1841188)
+- Tiebreaker
+  - **[Sewerslvt - Mr. Kill Myself [Darkness]](https://osu.ppy.sh/beatmapsets/1105241#osu/2309955)**
+
+### Qualifiers
+
+- NoMod
+  - [Ogura Yui - Honey Come!! [Simple]](https://osu.ppy.sh/beatmapsets/374144#osu/819641)
+  - [Festa - Lemuria [Nayarabbit's Expert]](https://osu.ppy.sh/beatmapsets/835474#osu/1806961)
+  - [EVERGLOW - Adios [Extra]](https://osu.ppy.sh/beatmapsets/1028706#osu/2174965)
+  - [IOSYS - Chanteikku Sanyousei no Itazura Daisensou [Crazy Jay]](https://osu.ppy.sh/beatmapsets/24448#osu/91462)
+- Hidden
+  - [Umeboshi Chazuke - Wakare no Namida wa Mata Au Hi made [Insane]](https://osu.ppy.sh/beatmapsets/830444#osu/1799281)
+  - [Warak - REANIMATE [Extreme]](https://osu.ppy.sh/beatmapsets/186318#osu/445177)
+- Hardrock
+  - [Megpoid GUMI - Shinkaron -code:variant- [Rin]](https://osu.ppy.sh/beatmapsets/29445#osu/99465)
+  - [Kana Nishino - Sweet Dreams (11t dnb mix) [Andrea's Insane]](https://osu.ppy.sh/beatmapsets/499488#osu/1114962)
+- DoubleTime
+  - [O-Life Japan - Yamanobori Kibun [Lunatic]](https://osu.ppy.sh/beatmapsets/30256#osu/99989)
+  - [Nitro Fun - Cheat Codes VIP [Light Insane]](https://osu.ppy.sh/beatmapsets/711420#osu/1506487)
+
+## Match results
+
+### Grand Finals
+
+Sunday, 2020-09-27:
```

Comment:
```suggestion
Sunday, 2020-09-27:
```
Use this format instead, for clearer reading
```Sunday, 27 September 2020```
Also, no match links available?

Reply:
prefer keeping date consistent w/ other instances of the tournament

will add match links though